padertorch/train/trainer.py

```diff
@@ -608,7 +624,7 @@ def _review_to_loss_and_summary(self, review):
 
         return loss, review
 
-    def log_error_state(self, data_dict, folder='log'):
+    def log_error_state(self, data_dict, folder='log', stdout=sys.stdout):
```

Comment:
I'd call this `file` (as in `print`) or `stream` or similar. `stdout` gets confusing if you set it to `stderr`.

Reply:
Good point. I changed it. I only introduced it to avoid capturing the stdout in the test. Since you now mention it, should the default be `stderr`? It is used to report problems in the failure logging progress.

Reply:
It's a little bit late now, but probably yes. The python `logging` module also uses `sys.stderr` as the default output stream.

Reply:
Yes, 1 second after I merged, I remembered this open point.
Ok, I will change it in the future.